src/test/kotlin/com/sainsburys/k2zpl/command/barcode/BarCode128Test.kt

```diff
@@ -0,0 +1,111 @@
+package com.sainsburys.k2zpl.command.barcode
+
+import com.sainsburys.k2zpl.command.options.ZplBarCode128Mode
+import com.sainsburys.k2zpl.command.options.ZplFieldOrientation
+import com.sainsburys.k2zpl.command.options.ZplYesNo
+import com.sainsburys.k2zpl.k2zpl
+import com.sainsburys.k2zpl.testBuildString
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.IsolationMode
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.data.forAll
+import io.kotest.data.headers
+import io.kotest.data.row
+import io.kotest.data.table
+import io.kotest.matchers.shouldBe
+
+class BarCode128Test : DescribeSpec({
+    isolationMode = IsolationMode.InstancePerLeaf
+
+    val subject = BarCode128(
+        orientation = ZplFieldOrientation.NORMAL,
+        height = 10,
+        line = ZplYesNo.NO,
+        lineAbove = ZplYesNo.NO,
+        checkDigit = ZplYesNo.NO,
+        mode = ZplBarCode128Mode.NONE
+    )
+    
+    describe("BarCode128") {
+        it("outputs correct command") {
+            val result = subject.testBuildString()
+            result shouldBe "^BCN,10,N,N,N,N"
+        }
+        it("uses orientation parameter properly") {
+            ZplFieldOrientation.entries.forEach {
+                subject.copy(orientation = it).testBuildString() shouldBe "^BC${it.code},10,N,N,N,N"
+            }
```

Comment:
Have we considered making this a table-driven test, and similarly for all other tests that use `.entries.forEach`?